src/components/cc-explore.vue

```diff
@@ -133,9 +134,16 @@
 import { defineComponent } from "vue";
 export default defineComponent({
   name: "cc-explore",
+  props: {
+    donation_url: {
+      type: String,
+      required: true,
+    },
+  },
   data() {
     return {
       isActive: false,
+      donation_link: this.donation_url,
```

Comment:
I believe the `donation_link` alias should be unnecessary since [we can access data passed into props directly in the template](https://vuejs.org/v2/guide/components.html#Passing-Data-to-Child-Components-with-Props).